### ISSUE TITLE
Doesn't poll for current brightness upon invocation.

### ISSUE THREAD

**Issue by roke-julian-lockhart** [JonMagon/plasma-screendimmer#2]
If I change my display brightness using https://github.com/davidhi7/ddcci-plasmoid

![image](https://github.com/JonMagon/plasma-screendimmer/assets/42837531/08a9ebb6-4415-435c-822f-92cfe22a6add)

and then switch to https://github.com/JonMagon/plasma-screendimmer

![image](https://github.com/JonMagon/plasma-screendimmer/assets/42837531/2d0c557d-a2d5-42fc-98fb-d3bfa7086945)

the wrong brightness is displayed.

**Comment by davidhi7** [comment]
I just randomly stumbled across this. My widget modifies the monitor brightness using DDC/CI, so basically on the hardware side, whereas this widget simply dims the picture on the software side. So since the widgets do different things, one won't affect the other.

**Comment by JonMagon** [comment]
Yes, the widgets are quite different. I don't see the point of using both together. I made this one mostly for myself, as my monitor doesn't support DDC/CI

**Comment by roke-julian-lockhart** [comment]
Actually, @davidhi7, yours only dims the software brightness for my monitor, an AOC Q3279WG5B. When I perform `systemctl poweroff`, the brightness returns to the value it was *before* https://github.com/davidhi7/ddcci-plasmoid set it during the shutdown sequence. https://github.com/JonMagon/plasma-screendimmer, to my knowledge.

**Comment by davidhi7** [comment]
This is really strange, I've never heard anything like this before. What monitor are we talking about? Also, feel free to open a bug report in my repository to address this.

**Comment by roke-julian-lockhart** [comment]
> I don't see the point of using both together.

@JonMagon, I use yours on my laptop because https://github.com/davidhi7/ddcci-plasmoid doesn't work by default (I expect due to https://github.com/davidhi7/ddcci-plasmoid/issues/21#issue-1686356497) whereas https://github.com/JonMagon/plasma-screendimmer does. Additionally, on my desktop, I can slide your widget's slider and the changes immediately update. The other, this is not true for.

**Comment by roke-julian-lockhart** [comment]
@davidhi7, https://github.com/JonMagon/plasma-screendimmer/issues/2#issuecomment-1636040125

> What monitor are we talking about?

@rokejulianlockhart, https://github.com/JonMagon/plasma-screendimmer/issues/2#issuecomment-1636025836:~:text=for%20my%20monitor%2C-,an%20aoc%20q3279wg5b,-.%20When%20I%20perform

> an AOC Q3279WG5B

---

> Also, feel free to open a bug report in my repository to address this.

Thanks. Done so - https://github.com/davidhi7/ddcci-plasmoid/issues/36#issue-1805346434.
